src/crimson/os/seastore/cache.cc

```diff
@@ -245,6 +246,77 @@ void Cache::register_metrics()
       ),
     }
   );
+
+  /*
+   * cache_query: cache_access and cache_hit
+   */
+  auto register_cache_access =
+    [this, &labels_by_src, &labels_by_ext](src_t src, extent_types_t ext) {
+      auto m_key = std::make_pair(src, ext);
+      stats.cache_query[m_key] = {0, 0};
+      std::ostringstream oss_labels;
+      oss_labels << "(src=" << src << ", ext=" << ext << ")";
+      auto str_labels = oss_labels.str();
```

Comment:
i'd avoid encoding labels into the description. just to minimize the repetition of information.

Reply:
All removed in this PR.